CondFormats/PhysicsToolsObjects/interface/Histogram3D.icc

```diff
@@ -71,7 +71,7 @@ namespace PhysicsTools {
           binULimitsX(binULimitsX),
           binULimitsY(binULimitsY),
           binULimitsZ(binULimitsZ),
-          binValues((binULimitsY.size() + 1) * strideX * strideY),
+          binValues((binULimitsZ.size() + 1) * strideX * strideY),
```

Comment:
@pieterdavid @ggovi for my understanding, this looks a bug fix, is it backward compatible in term of CondFormat? It is not changing neither the structure nor the interface of the format itself, so I would say it is...

Reply:
I assumed nobody used this constructor before, because if used to create a histogram with more Z than Y-bins, accessing a bin beyond the array would throw an exception (as long as this is not the constructor used by the deserialisation code it should be compatible).